packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts

```diff
@@ -28,19 +26,10 @@ export abstract class AnnotationScene extends _Scene.Group {
         this.toggleHandles(hovered);
     }
 
-    protected getCachedBBoxWithoutHandles() {
-        return this.cachedBBoxWithoutHandles ?? _Scene.BBox.zero;
-    }
-
     protected computeBBoxWithoutHandles() {
         return _Scene.Transformable.toCanvas(
             this,
             _Scene.Group.computeChildrenBBox(this.children.filter((node) => !(node instanceof Handle)))
         );
     }
-
-    override render(renderCtx: _Scene.RenderContext) {
-        super.render(renderCtx);
-        this.cachedBBoxWithoutHandles = this.computeBBoxWithoutHandles();
-    }
```

Comment:
Just to double check, this was added to stop the anchor point (and therefore the floating options buttons) from shifting after you first select an annotation and then the next render changed the bbox.

Reply:
I'll double check the use-case, but this code was not being used so I don't believe it is currently having any effect.

Reply:
Specifically, `getCachedBBoxWithoutHandles()` is not called anywhere, so the caching is not being used?

Reply:
Discussed offline - agreed this code was redundant.